### PR TITLE
paturage: Don't use shadow layers anymore

### PR DESCRIPTION
**Pull request by obiwac**
This is to make depth buffer a little tighter for devices for which WebGL provides 16-bit buffers (like Samsung phones, sometimes).

I realize now that this won't work, because some shadows are rendered *after* some cows, so, since we're not doing depth testing for them anymore, they'll draw *over* those cows!

The proper solution here would be to render all shadows before rendering all cows.

Don't forget to fix this [here](https://obiw.ac/moodle) too.

image

### CODE REVIEW

**Comment by obiwac** [comment]
how

**Comment by obiwac** [comment]
it was hacky before, sure, but now... how

File: public/static/js/paturage.js [hunk 1]
```diff
@@ -353,13 +353,17 @@ class Cow {
 		// yeah this is a bit of a hacky way to do shadows, but it works okay!
```
[the remark this answers is not in the document]
Comment (obiwac) [reply public/static/js/paturage.js:353]: oh, you're right. can you make a review with a suggestion to remove that comment so we can fix it?

[the remark this answers is not in the document]
Comment (obiwac) [reply public/static/js/paturage.js:353]: thanks!

**Review by Alexisloic21 — APPROVED** [review]

**Comment by obiwac** [comment]
il est pas aussi gentil en privé

**Pull request merged**